platform/features/conductor/src/ConductorRepresenter.js

```diff
@@ -97,12 +101,10 @@ define(
             }
 
             function updateDomain(value) {
-                conductor.domain(value);
-                repScope.$broadcast('telemetry:display:bounds', bounds(
-                    conductor.displayStart(),
-                    conductor.displayEnd(),
-                    conductor.domain()
-                ));
+                var newDomain = conductor.domain(value);
+                conductorScope.parameters.format =
+                    newDomain && newDomain.format;
```

Comment:
Similar to above-- is it valid for conductor.domain() not to return a domain object?  (isn't this part of the definition of `conductor.domain`)?